library/src/main/java/com/cube/storm/ui/view/holder/list/StandardListItemViewHolder.java

```diff
@@ -77,5 +79,17 @@ public StandardListItemViewHolder(View view)
 				}
 			});
 		}
+		
+		if (chevron != null)
```

Comment:
Can't remember how `populateView` is used, but could there be potential reuse issues with an `if` like this when the viewHolders bind

Reply:
What potential re-use issues do you mean?
Afaik the ViewHolder is bound to a view once (when created), and then `populateView` is called every time the view needs updating with a new model.
I included this `if (chevron != null)` since Storm apps with custom `standard_list_item_view.xml` resources may not necessarily include the view `R.id.chevron`, and I didn't want a null pointer exception in the case that they don't.

Reply:
Potentially I'm bringing over iOS `UITableView` stuff that doesn't apply here.

Because I know on both when you scroll you'll get reused view holders for rows that went off screen and you have to update them with new data. But don't think it has the same reuse implications as iOS.